enterprise_access/apps/bffs/tests/test_handlers.py

```diff
@@ -172,21 +172,58 @@ def test_load_and_process_without_learner_portal_enabled(self, mock_get_enterpri
         Test load_and_process method without learner portal enabled. No enterprise
         customer metadata should be returned.
         """
+        mock_customer_without_learner_portal = {
+            **self.mock_enterprise_customer,
+            'enable_learner_portal': False,
+        }
         mock_get_enterprise_customers_for_user.return_value = {
             **self.mock_enterprise_learner_response_data,
-            'results': [{
-                **self.mock_enterprise_customer,
-                'enable_learner_portal': False,
-            }],
```

Comment:
[inform] This mocked data was incorrect; the `results` is a list of `EnterpriseCustomerUser` records, not `EnterpriseCustomer`.